.github/workflows/requirements-txt-automatically-generated.yaml

```diff
@@ -0,0 +1,30 @@
+name: requirements.txt automatically generated
+
+on:
+  push:
+    branches: [master]
+
+  pull_request:
+    paths:
+      - '.github/workflows/requirements-txt-automatically-generated.yaml'
+
+jobs:
+  requirements-generated-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+
+      - name: Install pip-tools
+        run: |
+          pip3 install pip-tools
+
+      - name: Run Tests
+        run: |
+          make test-requirements
```

Comment:
You need to create this target. Please name it `test-requirements-txt-automatically-generated`. 
Also, follow the same pattern as in https://github.com/kubeflow/pipelines/pull/11509, with a [setup target](https://github.com/kubeflow/pipelines/pull/11509/files#diff-b224dfdea0489e673f571b5ff2c821aadf154a630918adac30e0258ce73f7ecaR29-R31). You'll have to create the `Makefile.setup.mk` file, in case that PR is not merged yet.

Reply:
thanks, i'll follow your guideline.